website/source/docs/runtime/environment.html.md

```diff
@@ -117,24 +125,28 @@ details.
 
 ### Task Directories
 
-Nomad makes the following two directories available to tasks:
+Nomad makes the following directories available to tasks:
 
 * `alloc/`: This directory is shared across all tasks in a task group and can be
   used to store data that needs to be used by multiple tasks, such as a log
   shipper.
 * `local/`: This directory is private to each task. It can be used to store
   arbitrary data that shouldn't be shared by tasks in the task group.
+* `secrets/`: This directory is private to each task, not accessible via the
+  `nomad fs` command or filesystem APIs and where possible backed by an
+  in-memory filesystem. It can be used to store secret data that shouldn't be
+  visible outside the task.
 
-Both these directories are persisted until the allocation is removed, which
-occurs hours after all the tasks in the task group enter terminal states. This
-gives time to view the data produced by tasks.
+These directories are persisted until the allocation is removed, which occurs
+hours after all the tasks in the task group enter terminal states. This gives
+time to view the data produced by tasks.
 
 Depending on the driver and operating system being targeted, the directories are
 made available in various ways. For example, on `docker` the directories are
 bound to the container, while on `exec` on Linux the directories are mounted into the
 chroot. Regardless of how the directories are made available, the path to the
-directories can be read through the `NOMAD_ALLOC_DIR` and `NOMAD_TASK_DIR`
-environment variables.
+directories can be read through the `NOMAD_ALLOC_DIR`, `NOMAD_TASK_DIR` and
```

Comment:
![](http://thewritepractice.com/wp-content/uploads/2011/08/Lets-eat-grandpa.jpg)

http://thewritepractice.com/why-you-need-to-be-using-oxford-commas/